x-pack/plugins/siem/public/management/pages/policy/view/ingest_manager_integration/index.tsx

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { ConfigureEndpointDatasource } from './configure_datasource';
```

Comment:
why? this will make it so that tsserver finds multiple valid import locations for the same thing

Reply:
ah sorry that was from a poc commit, i removed the additional import